packages/common/src/entities/environments/environment.local.ts

```diff
@@ -56,6 +56,9 @@ export class LocalEnvironment extends EnvironmentAbstract {
             case ENTITY_TYPES.PROJECT:
                 return path.join(this.dirPaths.projectsData, `${ENTITY_TYPES.PROJECT}-${id}`);
 
+            case ENTITY_TYPES.PROJECT_INSTALL:
+                return path.join(this.dirPaths.projectsData, `${ENTITY_TYPES.PROJECT}-${id}`);
```

Comment:
I'm guessing this is meant to be:

```suggestion
                return path.join(this.dirPaths.projectsData, `${ENTITY_TYPES.PROJECT_INSTALL}-${id}`);
```

Reply:
No, I made that mistake too. But the project id that a relate.project-install.json is located in is that one.